dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java

```diff
@@ -209,8 +196,7 @@ public String getConfigs(String key, String group, long timeout) throws IllegalS
     @Override
     public Object getInternalProperty(String key) {
         try {
-            String[] keyAndGroup = getKeyAndGroup(key);
-            return configService.getConfig(keyAndGroup[0], keyAndGroup[1], 5000L);
+            return configService.getConfig(key, null, DEFAULT_TIMEOUT);
```

Comment:
why ignore the  group specified by user

Reply:
The null value should be change to "dubbo" instead.